docker/mcp/services.py

```
@@ -53,19 +53,28 @@ def fetch_search_results(query: str, api_key: str) -> List[Dict[str, Any]]:
     }
     
-    response = requests.post(serper_url, headers=headers, json=payload)
-    search_results = response.json()
-    
-    if 'organic' not in search_results or not search_results['organic']:
-        return []
+    try:
+        response = requests.post(serper_url, headers=headers, json=payload)
+        response.raise_for_status()  # Raise exception for 4XX/5XX status codes
+        search_results = response.json()
         
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)